hooks/persistence-static-report/README.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Static Report"
+path: "hooks/persistence-staticreport"
+category: "hook"
+type: "persistenceProvider"
+state: "developing"
+usecase: "Publishes all Scan Findings as HTML Report."
+---
+
+<!-- end -->
+
+## Deployment
+The secureCodeBox core team is working on an integration of DefectDojo. We will keep you informed.
```

Comment:
Copy Paste from DD

```suggestion
The implementation is currently work-in-progress and under still undergoing major changes. It'll be released here once it has stabilized.
```

Reply:
The messages about the develop status are now different.